src/defaults.js

```diff
@@ -0,0 +1,82 @@
+export default {
+    ignoreRouteUrls: [
+        '/api/health',
+        '/gql/health',
+    ],
+    console: {
+        colorize: false,
+    },
+    level: 'info',
+    loggly: {
+        enabled: false,
+        environment: 'dev',
+        tagName: 'leif', // xxx set tag name properly
```

Comment:
I need to fix this. Should be easy enough to set this properly from the logger factory.